src/main/java/io/stargate/sgv2/jsonapi/service/operation/reranking/Feature.java

```diff
@@ -0,0 +1,43 @@
+package io.stargate.sgv2.jsonapi.service.operation.reranking;
+
+import java.util.Objects;
+
+/**
+ * Represents distinct features that can be used in the command. Each feature has an associated tag
+ * name, which can be used for metrics.
+ */
+public enum Feature {
```

Comment:
Name "Feature" feels very generic... should probably be something more specific as we also have `ApiFeature`? If this is specific to reranking, perhaps `RerankingFeature`?
If more general, `CommandFeature`?

Reply:
the idea was to be general, maybe CommandFeature is a good idea - most of what we do is command based